.github/workflows/build-binaries.yml

```diff
@@ -1,12 +1,22 @@
 name: Build Core binaries
-on: workflow_dispatch
+on:
+  workflow_dispatch:
+    inputs:
+      core-version:
+        type: string
+        required: false
+        default: ''
+        description: Core version (tag/ref) to use to generate the binaries for. If not provided, the version in dependencies.list will be used
```

Comment:
(tag/ref) implies that you can build things other than tagged versions with this but I'm pretty sure doing so will fail as `REALM_VERSION` needs to be something that can be parsed as a version. To support that it'd need to do something along the lines of checking out the supplied branch but using the result of `git describe` as the version everywhere else.

Reply:
I've clarified this now. This should either be a tag or the output of `git describe`. It doesn't appear like we can do actual releases with a `git describe`-d version, but we can still create prebuilds and use them during development if we wanted to run a CI build on all platforms from an untagged version of Core.